scan/src/pages/ValidatorHomePage.re

```diff
@@ -91,6 +161,61 @@ let getCurrentDay = _ => {
   MomentRe.momentNow() |> MomentRe.Moment.format(Config.timestampUseFormat);
 };
 
+module SortableDropdown = {
+  [@react.component]
+  let make = (~sortedBy, ~setSortedBy) => {
+    let (show, setShow) = React.useState(_ => false);
+    let sortList = [
+      ValidatorsTable.NameAsc,
+      NameDesc,
+      VotingPowerAsc,
+      VotingPowerDesc,
+      CommissionAsc,
+      CommissionDesc,
+      UptimeAsc,
+      UptimeDesc,
+    ];
+    <div className=Styles.sortDrowdownContainer>
+      <div className=Styles.sortDropdownTextItem onClick={_ => setShow(prev => !prev)}>
+        <Text
+          block=true
+          value={ValidatorsTable.getName(sortedBy)}
+          size=Text.Md
+          weight=Text.Regular
+          color=Colors.gray6
+          align=Text.Right
+        />
+      </div>
+      <div className={Styles.sortDrowdownPanel(show)}>
+        {sortList
+         ->Belt.List.map(value => {
+             let isActive = sortedBy == value;
+             <div
+               className={Styles.sortDropdownItem(isActive)}
+               onClick={_ => {
+                 setSortedBy(_ => value);
+                 setShow(_ => false);
+               }}>
```

Comment:
Please add key here.

Reply:
I'm not sure I can whether use the` index` of mapWithIndex or not?